### DISCUSSION TITLE
Allow endpoint switches to reside outside work area.

### DISCUSSION THREAD

**Discussion by i-g-g-y** [winder/Universal-G-Code-Sender#1857] [Help]
### Description

Hello.

I would like the end switches to reside outside the work area. Right now the work area includes <0;0;0>, meaning that if we want to use whole area, the switch will trigger. I know this is a bit lame, as we never use the entire work area, but it gets annoying when jogging and hitting the limit switch in work space.

**Comment by breiler** [comment]
This sounds like something that should be configured in the controller or where your physical limit switches are placed. 
UGS is dumb in this regard, just reporting the current controller position and relying on the firmware to throw errors if any movement is considered valid or dangerous.

**Comment by i-g-g-y** [comment]
I undrstand. Thank you!
